main/src/modules/humanUnderstanding/agentDetector/AgentDetector.cpp

```diff
@@ -450,58 +431,58 @@ bool AgentDetector::updateModule()
             bPresent.addString("==");
             bPresent.addInt(1);
 
-            bCond.addList() = bObject;
+            bCond.addList()=bObject;
             bCond.addString("&&");
-            bCond.addList() = bPresent;
+            bCond.addList()=bPresent;
             bCond.addString("||");
-            bCond.addList() = bRTObject;
+            bCond.addList()=bRTObject;
             bCond.addString("&&");
-            bCond.addList() = bPresent;
+            bCond.addList()=bPresent;
             opc->checkout();
-            opc->isVerbose = true;
-            list<Entity*> presentObjects = opc->Entities(bCond);
-            opc->isVerbose = false;
-            if (presentObjects.size() != 1)
+            opc->isVerbose=true;
+            list<Entity*> presentObjects=opc->Entities(bCond);
+            opc->isVerbose=false;
+            if (presentObjects.size()!=1)
             {
                 cout<<"There should be 1 and only 1 object on the table"<<endl;
             }
             else
             {
-                Object* o = (Object*) (presentObjects.front());
+                Object* o=(Object*)(presentObjects.front());
                 //Prepare the bottle to be sent to RFH
                 Bottle botRPH, botRPHRep;
                 botRPH.addString("add");
                 botRPH.addString("kinect");
-                Bottle &cooKinect = botRPH.addList();
+                Bottle &cooKinect=botRPH.addList();
                 cooKinect.addDouble(clickedPoint[0]);
                 cooKinect.addDouble(clickedPoint[1]);
                 cooKinect.addDouble(clickedPoint[2]);
 
-                Bottle &cooiCub = botRPH.addList();
+                Bottle &cooiCub=botRPH.addList();
                 cooiCub.addDouble(o->m_ego_position[0]);
                 cooiCub.addDouble(o->m_ego_position[1]);
                 cooiCub.addDouble(o->m_ego_position[2]);
                 rfh.write(botRPH,botRPHRep);
                 cout<<"Sent to RFH: "<<botRPH.toString().c_str()<<endl;
                 cout<<"Got from RFH: "<<botRPHRep.toString().c_str()<<endl;
-
-                pointsCount++;
-                if(pointsCount >= 3 )
```

Comment:
Shall we keep this check? I.e. `else if (AgentDetector::clicked==clicked_right && pointsCount >=3)`. Otherwise we should check whether the calibration was really successful.